services/api/tests/src/test_rsu_geo_query.py

```diff
@@ -93,3 +93,19 @@ def test_query_rsu_devices(mock_query_db):
 
     assert actual_result == ["10.11.81.12"]
     assert code == 200
+
+@patch("api.src.rsu_commands.pgquery.query_db")
+def test_query_rsu_devices_with_vendor(mock_query_db):
+    mock_query_db.return_value = [
+        ({"ip": "10.11.81.12"},),
+    ]
+    print(rsu_geo_query_data.point_list)
```

Comment:
Is this print statement necessary?

Reply:
Nope - I've gone ahead and removed this. Thanks for catching that